src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -21,15 +21,14 @@ def count_flops_params(model: nn.Module, input_size, verbose=True):
     the remained filters according to its mask, which
     not taking the pruned input channels into consideration,
     so the calculated FLOPs will be larger than real number.
-
     Parameters
     ---------
     model : nn.Module
         target model.
     input_size: list, tuple
         the input shape of data
-
-
+    custom_ops:
```

Comment:
what is the type of `custom_ops`? it should be an nn.Module?

Reply:
It should be `dict`, like `custom_mask_ops `

Reply:
please explain the type and how to use it in docstring